examples/performance/basic.e

```diff
@@ -0,0 +1,81 @@
+class
+	BASIC
+
+create
+	make
+
+feature {NONE} -- Initialization
+
+	make
+			-- Initialize `Current'.
+		local
+			parser: JSON_PARSER
+			l_stopwatch: DT_STOPWATCH
+			i: INTEGER
+		do
+			create file_reader
+				-- Create parser for content `json_content'
+			if attached json_file_from ("json60mb.json") as json_content then
```

Comment:
instead of having the file json60mb.json in the repository .. could we "generate" it ?
it would make the git repository smaller...